index.html

```diff
@@ -227,7 +227,20 @@ <h3>
         </ul>
       </div>
     </section>
-
+    <section id="contact">
+      <footer>
+          <h2>Contact me</h2>
+          <p id="para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          <form action="#" method="post">
+            <input class="name" type="text" name="user-name" placeholder="Write Your Name Here">
+            <input class="email" type="text" name="user-name" placeholder="Write Your Email Here">
+            <textarea  class="textarea" name="comments" id="name" cols="30" rows="10" placeholder="Write your message here"></textarea>        
+            <input class="button" type="submit" value="Get in Touch">
```

Comment:
- [ ] Following the project requirements, All input fields should have the required attribute to activate the validation on submit. Kindly add it 😃.

Reply:
> * [ ]  Following the project requirements, All input fields should have the required attribute to activate the validation on submit. Kindly add it 😃.

- [ ] Corrected 💪🏻

Reply:
hi @BenMukebo  the requested changes applied you can check and approve it because i got 3 reviews . one thing that im not understanding is that these changes which the reviewers ask is already done im checked in vs code run in browser all ok  and pushed to github but again they asking the same thing